### PR TITLE
chore: change examples calls from generateAuthToken to generateApiKey

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

File: examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts [hunk 1]
```diff
@@ -34,13 +34,13 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
...
-interface VendedAuthToken {
-  authToken: string;
+interface VendedApiKey {
+  apiKey: string;
   expiresAt: number;
 }
 
-async function vendAuthToken(vendorAuthTokenSecretName: string, headers: APIGatewayProxyEventHeaders): Promise<VendedAuthToken> {
-  const momentoAuthClient = await getMomentoAuthClient(vendorAuthTokenSecretName);
+async function vendApiKey(vendorApiKeySecretName: string, headers: APIGatewayProxyEventHeaders): Promise<VendedApiKey> {
```
Comment (anitarua) [inline examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts:42]: should it be named `vendDisposableToken` instead if the TVM is using `generateDisposableToken`?

File: examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts [hunk 2]
```diff
@@ -34,13 +34,13 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
...
 };
 
-interface VendedAuthToken {
-  authToken: string;
+interface VendedApiKey {
+  apiKey: string;
   expiresAt: number;
 }
 
-async function vendAuthToken(vendorAuthTokenSecretName: string, headers: APIGatewayProxyEventHeaders): Promise<VendedAuthToken> {
```
Comment (krispraws) [inline examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts:42]: We are vending disposable tokens, so we shouldn't rename this.

File: examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts (hunk 1 above)
Comment (cprice404) [reply examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts:42]: nice catch, thanks!

File: examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts (hunk 2 above)
Comment (cprice404) [reply examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts:42]: you and @anitarua - great minds think alike! thanks for catching this

Comment (cprice404) [reply examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts:42]: fixed in https://github.com/momentohq/client-sdk-javascript/pull/877/commits/a0d8beddd9a32923612fd466851b9c00198832d7

File: examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts (hunk 1 above)
Comment (cprice404) [reply examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts]: fixed in https://github.com/momentohq/client-sdk-javascript/pull/877/commits/a0d8beddd9a32923612fd466851b9c00198832d7

**Review by anitarua — APPROVED** [review]

**Review by krispraws — APPROVED** [review]

**Pull request merged**